sickbeard/processTV.py

```diff
@@ -665,3 +670,25 @@ def subtitles_enabled(video):
     else:
         logger.log(u'Empty indexer ID for: {}'.format(video), logger.WARNING)
         return
+ 
+def has_embedded_subtitles(video):
```

Comment:
One idea: Instead of duplicating this code, just use the video filename to create a subliminal video using subliminal API.
Subliminal will return you a video with this information already set (since it will run the metadata refiner). Then you should just check if the subtitle/embedded is present.